.github/workflows/quickstart_1.0.yml

```diff
@@ -12,18 +12,18 @@ on:
 
 jobs:
   get_input-srpms:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
     - name: Checkout
       uses: actions/checkout@v3
       with:
         ref: '1.0-stable'
 
-    - name: Set up Go 1.17
+    - name: Set up Go 1.18
```

Comment:
the name fields in these files need updating as well

Reply:
It should be fine now, thanks.

Reply:
wait shouldn't it be 1.19 instead of 1.18?

Reply:
Oh, sorry, I missed that one, I focused on the `go-version` fields.:/ It should be fine now.